addons/godot-next/global/physics_layers.gd

```diff
@@ -0,0 +1,59 @@
+# PhysicsLayers
+# author: xaguzman
+# license: MIT
+# description: A Utility class which allows easy access to your physics layers via their names in the project settings.
+tool
+extends Node
```

Comment:
For some reason, marking it as a Resource would  not work before, it would warn me that the class needed to be a Node or something...I forgot.
I was eventually just able to make it work.